### ISSUE TITLE
UNUSED_LOCAL_VARIABLE false positive with var used in cfmail(){} in cfscript

### ISSUE THREAD

**Issue by Adysone** [cflint/CFLint#643]
I'm getting this false positive using CFLint 1.4.0.

```
public void function test()
{
	var _mail = "oui";
	cfmail()
	{
		writeOutput( _mail );
	}
}
```

`cflint: warning
INFO - Local variable _mail is not used in function test. Consider removing it.`

**Comment by KamasamaK** [comment]
Probably duplicate of #593

Confirmed fixed for v1.5.0
